<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Revrsi</class>
 <widget class="QMainWindow" name="Revrsi">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>808</width>
    <height>655</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Revrsi</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QGraphicsView" name="graphicsView"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QLabel" name="spieler_farbe">
          <property name="text">
           <string>Aktueller Spieler</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="Akt_Spieler_Label">
          <property name="text">
           <string>Schwarz</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>808</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuSpiel">
    <property name="title">
     <string>Spiel</string>
    </property>
    <addaction name="actionNeu"/>
    <addaction name="actionBeenden"/>
   </widget>
   <widget class="QMenu" name="menuNetzwerk">
    <property name="title">
     <string>Netzwerk</string>
    </property>
    <addaction name="actionServer"/>
    <addaction name="actionClient"/>
   </widget>
   <widget class="QMenu" name="menuHilfe">
    <property name="title">
     <string>Hilfe</string>
    </property>
    <addaction name="actionHilfe"/>
    <addaction name="action_ber"/>
   </widget>
   <addaction name="menuSpiel"/>
   <addaction name="menuNetzwerk"/>
   <addaction name="menuHilfe"/>
  </widget>
  <action name="actionNeu">
   <property name="text">
    <string>Neu</string>
   </property>
  </action>
  <action name="actionBeenden">
   <property name="text">
    <string>Beenden</string>
   </property>
  </action>
  <action name="actionOptionen">
   <property name="text">
    <string>Optionen</string>
   </property>
  </action>
  <action name="actionServer">
   <property name="text">
    <string>Server</string>
   </property>
  </action>
  <action name="actionClient">
   <property name="text">
    <string>Client</string>
   </property>
  </action>
  <action name="action_ber">
   <property name="text">
    <string>Ãœber</string>
   </property>
  </action>
  <action name="actionHilfe">
   <property name="text">
    <string>Hilfe</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>actionOptionen</sender>
   <signal>triggered()</signal>
   <receiver>Revrsi</receiver>
   <slot>show()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>403</x>
     <y>327</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
